app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -188,23 +184,16 @@ def populate_hardware_hash_with_series_attributes(hardware_hash, instance, serie
         return if series.nil?
         hardware_hash[:logical_cpus]  = series[:cpus]
         hardware_hash[:memory_cpu]    = series[:memory] # MB
-        hardware_hash[:disk_capacity] = series[:os_disk_size] + series[:swap_disk_size]
+        hardware_hash[:disk_capacity] = series[:root_disk_size] + series[:swap_disk_size]
 
         os_disk = instance.fetch_path('properties', 'storageProfile', 'osDisk')
-        sz      = series[:os_disk_size]
+        sz      = series[:root_disk_size]
 
         add_instance_disk(hardware_hash[:disks], sz, os_disk['name'], os_disk['vhd']) unless sz.zero?
 
         # No data availbale on swap disk? Called temp or resource disk.
```

Comment:
@Fryguy @jvlcek 
Swap disk size is available but nothing more, this why we did not create an instance disk for swap disk. However, maybe we can create a bare bones disk in the DB… Openstack did so. Not sure if that is relevant to this PR though.